common/src/KokkosFFT_utils.hpp

```diff
@@ -8,19 +8,60 @@
 #include <Kokkos_Core.hpp>
 #include <vector>
 #include <set>
+#include <string>
 #include <algorithm>
 #include <numeric>
 #include "KokkosFFT_traits.hpp"
 
+#ifndef KOKKOS_ENABLE_CXX17
+#include <source_location>
+#define KOKKOSFFT_EXPECTS(expression, msg)             \
+  KokkosFFT::Impl::check_precondition(expression, msg, \
+                                      std::source_location::current())
+#else
+#include <cstdlib>
+#define KOKKOSFFT_EXPECTS(expression, msg)                                 \
+  KokkosFFT::Impl::check_precondition(expression, msg, __FILE__, __LINE__, \
```

Comment:
I suspect you need the extraneous parentheses
```suggestion
  KokkosFFT::Impl::check_precondition((expression), msg, __FILE__, __LINE__, \
```

Reply:
Fixed accordingly